apps/api/src/app/workflows/usecases/get-workflow-override/get-workflow-override.usecase.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+
+import { TenantRepository, WorkflowOverrideRepository } from '@novu/dal';
+import { GetWorkflowOverrideResponseDto } from '../../dto/get-workflow-override-response.dto';
+import { GetWorkflowOverrideCommand } from './get-workflow-override.command';
+
+@Injectable()
+export class GetWorkflowOverride {
+  constructor(
+    private tenantRepository: TenantRepository,
+    private workflowOverrideRepository: WorkflowOverrideRepository
+  ) {}
+
+  async execute(command: GetWorkflowOverrideCommand): Promise<GetWorkflowOverrideResponseDto> {
+    const tenant = await this.tenantRepository.findOne({
+      _environmentId: command.environmentId,
+      identifier: command.tenantIdentifier,
+    });
+
+    if (!tenant) {
+      throw new NotFoundException(`Tenant with identifier ${command.tenantIdentifier} is not found`);
+    }
```

Comment:
if we will have `tenantId` then we don't need this

Reply:
Do you think we should switch the Param from `tenantIdentifier` to `tenantId` in the route? 
Please see https://github.com/novuhq/novu/pull/4729#discussion_r1382541366
I wonder what should be the interface of the ids in the route, should we use the `identifiers` or `_ids`